CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ if(NOT DEFINED PROJECT_NAME)
     set(LA_IS_PRIMARY_PROJECT ON)
 endif()
 
-project(wg21_linear_algebra VERSION 0.7.0)
+project(wg21_linear_algebra VERSION 0.7.2)
```

Comment:
Once Travis is fixed up I'll be releasing a 0.7.2 package, as this will form the baseline use for creating an official Conan package of Linear Algrbra here: https://github.com/conan-io/conan-center-index/pull/5700